src/libstd/net/ip.rs

```diff
@@ -244,6 +243,20 @@ impl FromInner<libc::in_addr> for Ipv4Addr {
     }
 }
 
+impl Into<u32> for Ipv4Addr {
```

Comment:
Due to the way `Into` and `From` works, this is more conventionally written as `impl From<Ipv4Addr> for u32`